io/disk/usb_tests.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2023 IBM
+# Author: Maram Srimannarayana Murthy <msmurthy@linux.vnet.ibm.com>
+"""
+This script will perform usb related testcases
+"""
+from avocado import Test
+from avocado.utils.software_manager.manager import SoftwareManager
+from avocado.utils import disk
+
+
+class USBTests(Test):
+
+    '''
+    Class to execute usb tests
+    '''
+
+    def setUp(self):
+        """
+        Function for preliminary set-up to execute the test
+        """
+        smm = SoftwareManager()
+        for pkg in ["usbguard"]:
+            if not smm.check_installed(pkg) and not smm.install(pkg):
+                self.cancel(f"{pkg} is not installed")
+        self.usb_pci_device = self.params.get("usb_pci_device", default="")
+        if not self.usb_pci_device:
+            self.cancel("please provide pci adrees or wwids of scsi disk")
```

Comment:
@maramsmurthy are you planning to use usb_pci_device  somewhere ? I dont see any where being used now

Reply:
Yes, We will be using this as part of usb guard test case